src/Movie/Movie.css

```diff
@@ -43,9 +44,9 @@
 }
 
 .poster-movie-view {
-    width: 13vw;
-    aspect-ratio: auto;
-    margin: 1.5vw;
+    align-self: center;
+    max-height: 75%;
```

Comment:
Added a boundary for the poster height so it isn't huge.

Reply:
love the use of max-height! these are huge image files lol